classes/user.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
I believe the names of class files are capitalized (User.php)

Reply:
Updated!